holoviews/core/data/__init__.py

```diff
@@ -516,13 +516,17 @@ def get_dimension_type(self, dim):
         return self.interface.dimension_type(self, dim_obj)
 
 
-    def dframe(self, dimensions=None):
+    def dframe(self, dimensions=None, copy=True):
```

Comment:
I'm not sure copy argument makes much sense if the element isn't already using a dataframe based interface - for other interfaces, don't you always have to create a new dataframe - which would be the same as ``copy`` being fixed to ``True``?